### ISSUE TITLE
Artisan commands not working in 6.0 RC3

### ISSUE THREAD

**Issue by Toreg87** [grokability/snipe-it#10704]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

After upgrading my test environment to 6.0 RC3 all artisan commands fail with following error:

Could not check compatibility between App\Exceptions\Handler::unauthenticated($request, App\Exceptions\AuthenticationException $exception) and Illuminate\Foundation\Exceptions\Handler::unauthenticated($request, Illuminate\Auth\AuthenticationException $exception), because class App\Exceptions\AuthenticationException is not available

The application is working fine however. I fixed this with inserting a "use Illuminate\Auth\AuthenticationException;" in app/Exceptions/Handler.php. I'm not sure this is the right fix. If so I can create a PR for this issue.

### Reproduction steps

1. Switched branch from master to develop
2. php artisan down

### Expected behavior

Artisan commands should work.

### Snipe-IT Version

6.0 RC3

### Operating System

Windows - XAMPP

### Web Server

Apache 2.4

### PHP Version

PHP 7.4

### Error messages

```shell
[2022-02-21 09:36:54] local.ERROR: ErrorException: Could not check compatibility between App\Exceptions\Handler::unauthenticated($request, App\Exceptions\AuthenticationException $exception) and Illuminate\Foundation\Exceptions\Handler::unauthenticated($request, Illuminate\Auth\AuthenticationException $exception), because class App\Exceptions\AuthenticationException is not available in C:\xampp\htdocs\snipe-it\app\Exceptions\Handler.php:114
Stack trace:
#0 C:\xampp\htdocs\snipe-it\app\Exceptions\Handler.php(14): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Could not check...', 'C:\\xampp\\htdocs...', 114, Array)
#1 C:\xampp\htdocs\snipe-it\vendor\composer\ClassLoader.php(571): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\snipe-it\vendor\composer\ClassLoader.php(428): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#3 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Exceptions\\...')
#4 [internal function]: spl_autoload_call('App\\Exceptions\\...')
#5 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(877): ReflectionClass->__construct('App\\Exceptions\\...')
#6 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(758): Illuminate\Container\Container->build('App\\Exceptions\\...')
#7 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(851): Illuminate\Container\Container->resolve('App\\Exceptions\\...', Array, false)
#8 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(296): Illuminate\Foundation\Application->resolve('App\\Exceptions\\...', Array, false)
#9 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(873): Illuminate\Container\Container->Illuminate\Container\{closure}(Object(Illuminate\Foundation\Application), Array)
#10 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(758): Illuminate\Container\Container->build(Object(Closure))
#11 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(851): Illuminate\Container\Container->resolve('Illuminate\\Cont...', Array, true)
#12 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Container\Container.php(694): Illuminate\Foundation\Application->resolve('Illuminate\\Cont...', Array)
#13 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(836): Illuminate\Container\Container->make('Illuminate\\Cont...', Array)
#14 C:\xampp\htdocs\snipe-it\vendor\nunomaduro\collision\src\Adapters\Laravel\CollisionServiceProvider.php(63): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#15 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(682): NunoMaduro\Collision\Adapters\Laravel\CollisionServiceProvider->register()
#16 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(75): Illuminate\Foundation\Application->register(Object(NunoMaduro\Collision\Adapters\Laravel\CollisionServiceProvider))
#17 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(659): Illuminate\Foundation\ProviderRepository->load(Array)
#18 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#19 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(237): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#20 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(310): Illuminate\Foundation\Application->bootstrapWith(Array)
#21 C:\xampp\htdocs\snipe-it\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(127): Illuminate\Foundation\Console\Kernel->bootstrap()
#22 C:\xampp\htdocs\snipe-it\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 {main}  
[2022-02-21 09:36:54] local.ERROR: Could not check compatibility between App\Exceptions\Handler::unauthenticated($request, App\Exceptions\AuthenticationException $exception) and Illuminate\Foundation\Exceptions\Handler::unauthenticated($request, Illuminate\Auth\AuthenticationException $exception), because class App\Exceptions\AuthenticationException is not available {"exception":"[object] (ErrorException(code: 0): Could not check compatibility between App\\Exceptions\\Handler::unauthenticated($request, App\\Exceptions\\AuthenticationException $exception) and Illuminate\\Foundation\\Exceptions\\Handler::unauthenticated($request, Illuminate\\Auth\\AuthenticationException $exception), because class App\\Exceptions\\AuthenticationException is not available at C:\\xampp\\htdocs\\snipe-it\\app\\Exceptions\\Handler.php:114)
[stacktrace]
#0 C:\\xampp\\htdocs\\snipe-it\\app\\Exceptions\\Handler.php(14): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError(2, 'Could not check...', 'C:\\\\xampp\\\\htdocs...', 114, Array)
#1 C:\\xampp\\htdocs\\snipe-it\\vendor\\composer\\ClassLoader.php(571): include('C:\\\\xampp\\\\htdocs...')
#2 C:\\xampp\\htdocs\\snipe-it\\vendor\\composer\\ClassLoader.php(428): Composer\\Autoload\\includeFile('C:\\\\xampp\\\\htdocs...')
#3 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Exceptions\\\\...')
#4 [internal function]: spl_autoload_call('App\\\\Exceptions\\\\...')
#5 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(877): ReflectionClass->__construct('App\\\\Exceptions\\\\...')
#6 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(758): Illuminate\\Container\\Container->build('App\\\\Exceptions\\\\...')
#7 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(851): Illuminate\\Container\\Container->resolve('App\\\\Exceptions\\\\...', Array, false)
#8 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(296): Illuminate\\Foundation\\Application->resolve('App\\\\Exceptions\\\\...', Array, false)
#9 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(873): Illuminate\\Container\\Container->Illuminate\\Container\\{closure}(Object(Illuminate\\Foundation\\Application), Array)
#10 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(758): Illuminate\\Container\\Container->build(Object(Closure))
#11 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(851): Illuminate\\Container\\Container->resolve('Illuminate\\\\Cont...', Array, true)
#12 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\Container.php(694): Illuminate\\Foundation\\Application->resolve('Illuminate\\\\Cont...', Array)
#13 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(836): Illuminate\\Container\\Container->make('Illuminate\\\\Cont...', Array)
#14 C:\\xampp\\htdocs\\snipe-it\\vendor\
unomaduro\\collision\\src\\Adapters\\Laravel\\CollisionServiceProvider.php(63): Illuminate\\Foundation\\Application->make('Illuminate\\\\Cont...')
#15 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(682): NunoMaduro\\Collision\\Adapters\\Laravel\\CollisionServiceProvider->register()
#16 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\ProviderRepository.php(75): Illuminate\\Foundation\\Application->register(Object(NunoMaduro\\Collision\\Adapters\\Laravel\\CollisionServiceProvider))
#17 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(659): Illuminate\\Foundation\\ProviderRepository->load(Array)
#18 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Bootstrap\\RegisterProviders.php(17): Illuminate\\Foundation\\Application->registerConfiguredProviders()
#19 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Application.php(237): Illuminate\\Foundation\\Bootstrap\\RegisterProviders->bootstrap(Object(Illuminate\\Foundation\\Application))
#20 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(310): Illuminate\\Foundation\\Application->bootstrapWith(Array)
#21 C:\\xampp\\htdocs\\snipe-it\\vendor\\laravel\\framework\\src\\Illuminate\\Foundation\\Console\\Kernel.php(127): Illuminate\\Foundation\\Console\\Kernel->bootstrap()
#22 C:\\xampp\\htdocs\\snipe-it\\artisan(35): Illuminate\\Foundation\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#23 {main}
"}
```

### Additional context

Upgrade

**Comment by mich-de** [comment]
for me the same:


`In Handler.php line 113:

  Could not check compatibility between App\Exceptions\Handler::unauthenticated($request, App\Exceptions\AuthenticationException $exception) and Illuminate\Foundation\Exceptions\Handler::unauthenticated($request, Illuminate\Auth\AuthenticationException $exception), because class App\Exceptions\AuthenticationExcep
  tion is not available`

**Comment by insert-waffle** [comment]
Same for me, tried running the key generation but have gotten the same error message.

**Comment by snipe** [comment]
Are you seeing this in RC4? I'm not able to reproduce this at all :(

**Comment by Toreg87** [comment]
This is fixed with 0a4a548f9cdaa0ca6d8f3615620f5aca593492a2 which is part of RC4.
